mp/src/game/server/baseentity.cpp

```diff
@@ -951,57 +950,6 @@ void CBaseEntity::DrawDebugGeometryOverlays(void)
 			NDebugOverlay::EntityBounds(this, 255, 255, 255, 0, 0 );
 		}
 	}
-	if ( m_debugOverlays & OVERLAY_AUTOAIM_BIT && (GetFlags()&FL_AIMTARGET) && AI_GetSinglePlayer() != NULL )
```

Comment:
You can also remove all references to FL_AIMTARGET since it's explicitly only used to mark "auto aim"-able entities.

There is also the `CAimTargetManager` class inside `entitylist.cpp` to remove.